src/Apps/Artist/Routes/AuctionResults/ArtistAuctionResults.tsx

```diff
@@ -102,47 +104,54 @@ class AuctionResultsContainer extends Component<
   }
 
   render() {
-    const auctionResultsLength = this.props.artist.auctionResultsConnection
-      .edges.length
-    const { totalCount } = this.props.artist.auctionResultsConnection
+    const { artist } = this.props
+    const auctionResultsLength = artist.auctionResultsConnection.edges.length
     return (
       <Subscribe to={[AuctionResultsState]}>
         {({ state }: AuctionResultsState) => {
           return (
             <>
               <Row>
+                {/* 
                 <Col sm={2} pr={[0, 2]}>
-                  <TableSidebar count={totalCount} />
+                   <TableSidebar count={totalCount} />
                 </Col>
-
-                <Col sm={10}>
-                  <TableColumns />
-
-                  <Box pt={0.5}>
-                    <Separator />
-                  </Box>
-
-                  <ArtistAuctionDetailsModal
-                    auctionResult={state.selectedAuction}
+                */}
+
+                {/* <Col sm={10}> */}
```

Comment:
✂️ ? also [here](https://github.com/artsy/reaction/pull/3118/files#diff-643ca957db8afbe569c78cea5a1bb9ffR154)? or you are keeping them here to be added back when the filters are added?